### PR TITLE
Contact page

### PR DESCRIPTION
**Pull request by NadaaFarook**
Issue #30

### CODE REVIEW

File: client/src/pages/contact.tsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+              Contact us at :
+            </Text>
+            <Heading
+              as="h1"
+              fontSize={['xl', '2xl', '4xl']}
+              py={[4, 2, 0]}
+              color={'brand.500'}
+              letterSpacing="tighter"
+            >
+              <a href="mailto:contact.neogcamp@gmail.com">
```
Comment (uditkumar01) [inline client/src/pages/contact.tsx:28]: don't use `anchor` tags use chakra `Link` or `NextLink` instead. Not mandatory for now

File: client/src/pages/contact.tsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+import NextLink from 'next/link'
+
+export default function Contact() {
+  return (
+    <Layout>
+      <Flex mt={8} flexDir={['column-reverse', 'column-reverse', 'row']}>
+        <Flex flexDir="column" justifyContent="space-between">
+          <Box p={[4, 5, 6]} pt={[8, 8, 32]} pb={8}>
+            <Heading as="h1" size="xl" color={'white'} letterSpacing="tighter">
+              Have any issue with the platform?
```
Comment (uditkumar01) [inline client/src/pages/contact.tsx:14]: Make a file and get these strings from there.

**Review by uditkumar01 — COMMENTED** [review]
Mentioned some minor fixes, please try to fix these.And rest if :100: :fire:

File: client/src/pages/contact.tsx (hunk 1 above)
Comment (sohamsshah) [reply client/src/pages/contact.tsx:28]: If using href, add target as "_blank" also to it.

**Comment by sohamsshah** [comment]
@NadaaFarook can you resolve the comments and conflicts, if any?

**Comment by MehulKChaudhari** [comment]
Good work there, LGTM

File: client/src/components/StatusCard/StatusCard.tsx [hunk 3]
```diff
@@ -16,29 +16,22 @@ export function StatusCard({ status, bgColor, step, submissionNo }: any) {
...
+          fontSize={['xl', '2xl', '3xl']}
+          color={theme.colors.brand['500']}
         >
           {' '}
+          Current Status :{' '}
           <Heading
             fontSize={['xl', '2xl', '3xl']}
-            color={theme.colors.brand['500']}
+            as="span"
+            color={status?.color == 'red' ? '#F13C3C' : theme.colors.white}
```
Comment (sohamsshah) [inline client/src/components/StatusCard/StatusCard.tsx:29]: just curious, shall we consider adding `#F13C3C` to our theme if not present?

File: client/src/pages/contact.tsx [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+              Contact us at :
+            </Text>
+            <Heading
+              as="h1"
+              fontSize={['xl', '2xl', '4xl']}
+              py={[4, 2, 0]}
+              color={'brand.500'}
+              letterSpacing="tighter"
+            >
+              <a target="_blank" href="mailto:contact.neogcamp@gmail.com">
```
Comment (sohamsshah) [inline client/src/pages/contact.tsx:28]: can we extract the mail href to a constant/string in data folder?

**Review by sohamsshah — CHANGES_REQUESTED** [review]
Everything else looks awesome to me.

File: client/src/pages/contact.tsx (hunk 4 above)
Comment (uditkumar01) [reply client/src/pages/contact.tsx:28]: +1 hard coding is not a good practice

Comment (MehulKChaudhari) [reply client/src/pages/contact.tsx:28]: Agreed it will also follow the consistency

**Pull request closed without merging**